src/Services/Utils/SQLUpgradeService.php

```diff
@@ -657,6 +657,43 @@ function upgradeFromSqlFile($filename, $path = '')
                 if ($skipping) {
                     $this->echo("<p class='text-success'>$skip_msg $line</p>\n");
                 }
+            } elseif (preg_match('/^#IfEyeFormLaserCategoriesNeeded/', $line)) {
+                $eyeFormCategoryParent = sqlStatementNoLog("SELECT `id`, `rght` FROM `categories` WHERE `name` = 'Eye Form'");
+                if (sqlNumRows($eyeFormCategoryParent) > 0) {
```

Comment:
Nice Steve however you know me I'm that guy! 
Doing this by sql is fine and Brady's method works but the better way imo is to make this function universal for all new categories. Allowing the Tree class do the appends the class will unsure the nodes fall inline and the tree will be cleaned up if needed. Just pragmatize the function.

I'd look into how new is added from the UI and clone that functionality 
If you want I will help.